### PR TITLE
Add June 7th blog post

### PR DESCRIPTION
**Pull request by azigler**
This PR adds a blog post entry for tomorrow, June 7th, about `mattermost-ai-framework`.

**Do not merge until June 7th.**

### CODE REVIEW

File: content/posts/2023-06-07-exploring-ai-in-strict-security.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Mattermost Open Sources Private Solution for Exploring Generative AI in Strict Security Environments"
+date: 2023-06-07T11:31:04+05:00
+draft: false
+author: Chen Lim
+github: chenilim
+community: ian.tien
```
Comment (chenilim) [inline content/posts/2023-06-07-exploring-ai-in-strict-security.md]: Should be chen.lim

**Review by chenilim — CHANGES_REQUESTED** [review]
LGTM - one minor change for my community alias.

**Comment by azigler** [comment]
@asaadmahmood I'm having a difficult time getting this to rebuild locally. It seems that `hugo -d docs` isn't using my latest changes, the outputs are the same. Do you mind taking a look?

**Comment by asaadmahmood** [comment]
@azigler Worked for me, I pushed the docs updated files to this PR.

**Review by chenilim — APPROVED** [review]

**Pull request closed without merging**